app/models/manageiq/providers/amazon/storage_manager/ebs/refresh_parser.rb

```diff
@@ -22,6 +22,18 @@ def ems_inv_to_hashes
     @data
   end
 
+  private
+
+  def parent_manager_fetch_path(collection, ems_ref)
+    @parent_manager_data ||= {}
+    return @parent_manager_data.fetch_path(collection, ems_ref) if @parent_manager_data.has_key_path?(collection,
+                                                                                                      ems_ref)
+
+    @parent_manager_data.store_path(collection,
+                                    ems_ref,
+                                    @ems.public_send(collection).try(:where, :ems_ref => ems_ref).try(:first))
+  end
+
   def get_volumes
```

Comment:
@gberginc - this method already exists in ```ManageIQ::Providers::Amazon::NetworkManager::RefreshParser```
Maybe it should be moved to ```ManageIQ::Providers::Amazon::RefreshHelperMethods```  which is already included in both the NetworkManager parser and this file.